static/beta/prod/navigation/rhel-navigation.json

```diff
@@ -1,318 +1,369 @@
-                "filterable": false,
-                "href": "https://access.redhat.com/products/subscription-central"
-              }
-            ]
-          },
-          {
-            "id": "imageBuilder",
-            "appId": "imageBuilder",
-            "title": "Image builder",
-            "href": "/insights/image-builder",
-            "product": "Red Hat Insights",
-            "description": "Build and manage Red Hat Enterprise Linux images and environments."
-          }
-        ]
-      },
-      {
-        "id": "registerSystems",
-        "appId": "registration",
-        "title": "Register Systems",
-        "filterable": false,
-        "href": "/insights/registration",
-        "product": "Red Hat Insights",
-        "description": "Register your systems with the Red Hat Insights Client to view them on the Red Hat Hybrid Cloud Console."
-      },
-      {
-        "title": "Toolkit",
-        "expandable": true,
-        "routes": [
+  "id": "rhel",
+  "title": "Red Hat Insights",
+  "navItems": [
+    {
+      "id": "overview",
+      "appId": "dashboard",
+      "title": "Dashboard",
+      "filterable": false,
+      "href": "/insights/dashboard",
+      "product": "Red Hat Insights"
+    },
+    {
+      "title": "Inventory",
+      "expandable": true,
+      "id": "inventory",
+      "description": "View details about your Red Hat Enterprise Linux systems.",
+      "routes": [
+        {
+          "id": "systems",
+          "appId": "inventory",
+          "title": "Systems",
+          "href": "/insights/inventory",
+          "product": "Red Hat Insights"
+        },
+        {
+          "id": "groups",
+          "appId": "inventory",
+          "title": "Groups",
+          "href": "/insights/inventory/groups",
+          "product": "Red Hat Insights",
+          "description": "Inventory groups",
+          "permissions": [
             {
-                "id": "remediations",
-                "appId": "remediations",
-                "title": "Remediations",
-                "href": "/insights/remediations",
-                "product": "Red Hat Insights",
-                "description": "Use Ansible Playbooks to resolve configuration, security, and compliance issues identified on your Red Hat Enterprise Linux systems."
+              "method": "featureFlag",
+              "args": ["hbi.ui.inventory-groups", true]
+            }
+          ]
+        },
+        {
+          "id": "imageBuilder",
+          "appId": "imageBuilder",
+          "title": "Images",
+          "href": "/insights/image-builder",
+          "product": "Red Hat Insights",
+          "description": "Build and manage Red Hat Enterprise Linux images and environments."
+        },
+        {
+          "title": "System Configuration",
+          "expandable": true,
+          "routes": [
+            {
+              "id": "rhc",
+              "appId": "connector",
+              "title": "Remote Host Configuration (RHC)",
```

Comment:
should this be lowercase `(rhc)`?

Reply:
It is in capital in the [mocks](https://www.sketch.com/s/550bd962-ca1f-42d8-8d89-d457527a059a/a/mPowdv7)